packages/template-retail-react-app/jest.config.js

```diff
@@ -12,7 +12,8 @@ module.exports = {
     ...base,
     moduleNameMapper: {
         ...base.moduleNameMapper,
-        '^react-router-dom(.*)$': '<rootDir>/node_modules/react-router-dom/index.js'
+        '^react-router-dom(.*)$': '<rootDir>/node_modules/react-router-dom/index.js',
+        "^react($|/.+)": "<rootDir>/node_modules/react$1"
```

Comment:
Is this something new required because we imported the `commerce-sdk-react-preview` package?

Reply:
I think this is something we will eventually have to do once we integrate the commerce-sdk-react. I had to do it here to get tests working. But given we are copying over the Page/Regions/Components into the template we wont need this.